samples/nrf9160/modem_shell/src/utils/mosh_print.c

```diff
@@ -141,6 +148,18 @@ void mosh_fprintf_valist(enum mosh_print_level print_level, const char *fmt, va_
 		break;
 	}
 
+#if defined(CONFIG_MOSH_CLOUD)
+	if (mosh_print_cloud_echo) {
+		static struct nrf_cloud_sensor_data mosh_cloud_print = {
+			.type = NRF_CLOUD_DEVICE_INFO,
+			.data.ptr = mosh_print_buf,
+			.data.len = sizeof(mosh_print_buf),
```

Comment:
This is the size of the buffer, not length of the string. Should this use `strlen()` instead?